frontend/src/app/search/search.component.ts

```diff
@@ -169,6 +171,15 @@ export class SearchComponent implements OnInit, AfterViewInit, OnDestroy {
     // tslint:disable-next-line:deprecation
     this.mobileQuery.addListener(this._mobileQueryListener);
     window.addEventListener('resize', this._mobileQueryListener);
+    this.meta.updateTag(
```

Comment:
This feels repetitive across the different components  - maybe a service class could be used to update the tags?

Reply:
Agreed. If I standardized how the images were named for all the hero content, it would be much easier to set this at the app.component level where I'm setting the titles. Worth considering the options as we fine-tune it with the team.